src/Microsoft.AspNetCore.Mvc.Abstractions/ModelBinding/ModelBindingContext.cs

```diff
@@ -68,15 +68,38 @@ public abstract class ModelBindingContext
         public abstract string ModelName { get; set; }
 
         /// <summary>
-        /// Gets or sets the <see cref="ModelStateDictionary"/> used to capture <see cref="ModelState"/> values
-        /// for properties in the object graph of the model when binding.
+        /// Gets or sets the <see cref="ModelStateDictionary"/> used to capture
+        /// <see cref="ModelBinding.ModelStateEntry"/> values for properties in the object graph of the model when
```

Comment:
Old comment referenced the `ModelState` property in this documentation of that property.